src/packages/cli/src/args.ts

```diff
@@ -281,9 +281,17 @@ export default function (
             }
           )
           .version(false);
+      },
+      function () {
+        // this handler executes when `ganache instances` is called without a subcommand
+        const command = chalk`{hex("${TruffleColors.porsche}") ganache instances}`;
+        console.log(`${command} requires a subcommand:`);
+        console.log();
+        yargs.showHelp();
+        yargs.exit(1, new Error("No subcommand provided"));
       }
     )
-    .showHelpOnFail(false, "Specify -? or --help for available options")
+    .showHelpOnFail(false)
```

Comment:
Why did the help text need to be deleted (aside: I can't figure out how to actually get it to show up anyway)

Reply:
Eh, I _think_ I'm fine with removing it, but if we keep it it won't change the above that you know of?

Here's the yargs code that uses this string: https://github.com/yargs/yargs/blob/2c144c4ea534646df26d6177f73ce917105c6c09/lib/usage.ts#L69

I couldn't think of a way we could trigger it to actually send this message. Can you? If not, i'm 👍 on this change and you can resolve this comment :-)

Reply:
Likewise